lib/elixir/lib/option_parser.ex

```diff
@@ -712,6 +707,7 @@ defmodule OptionParser do
 
   defp to_existing_key(option, true),
     do: String.to_atom(option)
+
```

Comment:
yes, but i thought we keep them together when the following clauses using the same two line definition,